src/analysis/convex/basic.lean

```diff
@@ -109,42 +109,42 @@ segment_symm 𝕜 y x ▸ left_mem_segment 𝕜 y x
 end mul_action_with_zero
 
 section module
-variables (𝕜) [module 𝕜 E]
+variables (𝕜) [module 𝕜 E] {x y z : E} {s : set E}
 
 @[simp] lemma segment_same (x : E) : [x -[𝕜] x] = {x} :=
 set.ext $ λ z, ⟨λ ⟨a, b, ha, hb, hab, hz⟩,
   by simpa only [(add_smul _ _ _).symm, mem_singleton_iff, hab, one_smul, eq_comm] using hz,
   λ h, mem_singleton_iff.1 h ▸ left_mem_segment 𝕜 z z⟩
 
-lemma mem_open_segment_of_ne_left_right {x y z : E} (hx : x ≠ z) (hy : y ≠ z)
-  (hz : z ∈ [x -[𝕜] y]) :
-  z ∈ open_segment 𝕜 x y :=
+lemma insert_endpoints_open_segment (x y : E) :
```

Comment:
Not sure about the name of that one. I think it's okay, but I would have expected some thing like `_left_right` instead of `_endpoints`.